packages/web3-eth-contract/test/integration/contract_deploy.test.ts

```diff
@@ -18,15 +18,25 @@ import { Contract } from '../../src';
 import { sleep } from '../shared_fixtures/utils';
 import { GreeterBytecode, GreeterAbi } from '../shared_fixtures/build/Greeter';
 import { DeployRevertAbi, DeployRevertBytecode } from '../shared_fixtures/build/DeployRevert';
-import { getSystemTestProvider, isWs, createTempAccount } from '../fixtures/system_test_utils';
+import {
+	getSystemTestProvider,
+	isWs,
+	createTempAccount,
+	createNewAccount,
+	signTxAndSendEIP2930,
+	signTxAndSendEIP1559,
+} from '../fixtures/system_test_utils';
 
 describe('contract', () => {
 	describe('deploy', () => {
 		let contract: Contract<typeof GreeterAbi>;
 		let deployOptions: Record<string, unknown>;
 		let sendOptions: Record<string, unknown>;
 		let acc: { address: string; privateKey: string };
-
+		let localAccount: { address: string; privateKey: string };
+		beforeAll(async () => {
+			localAccount = await createNewAccount({ refill: true });
```

Comment:
Tests with local wallet requirement shouldn't add account on node wallet: 
https://github.com/web3/web3.js/blob/d43ac3a96a74a8760e92c2993049d26131428c98/scripts/system_tests_utils.ts#L207

same change required here and in other test files.